doc/api/os.md

```diff
@@ -174,13 +127,11 @@ added: v0.9.4
 
 * Returns: {string}
 
-The `os.endianness()` method returns a string identifying the endianness of the
-CPU *for which the Node.js binary was compiled*.
+Returns a string identifying the endianness of the CPU for which the Node.js
+binary was compiled. This may be different than the endianness of the CPU on
+which the binary is running.
```

Comment:
I guess the `endianness` is always attached to the host. IMO the second statement brings in confusion

Reply:
Yeah, it seems wrong to me even though I added it. I couldn't figure out what else could possibly be implied by the emphasis (italics) of _for which the Node.js binary was complied_ in the original text. I'll remove both the emphasis and the text I added, which should resolve the issue.

Reply:
Resolved in https://github.com/nodejs/node/pull/30102/commits/4c0313591db7033ab4d209e4059da1f242259f34